units/console-getty.service.m4.in

```diff
@@ -9,12 +9,14 @@
 Description=Console Getty
 Documentation=man:agetty(8) man:systemd-getty-generator(8)
 After=systemd-user-sessions.service plymouth-quit-wait.service
-ConditionPathExists=/dev/console
 m4_ifdef(`HAVE_SYSV_COMPAT',
 After=rc-local.service
 )m4_dnl
 Before=getty.target
 
+# OCI containers may be run without a console
+ConditionPathExists=/dev/console
```

Comment:
note that console-getty.service as more uses than just containers. The idea is that it may be used as alternative to the whole VC/logind stuff, if all you need is a console on /dev/console, even on physical devices.